exercises/grade-school/canonical-data.json

```
@@ -27,6 +27,19 @@
           },
           "expected": []
         },
+        {
+          "uuid": "a0c7b9b8-0e89-47f8-8b4a-c50f885e79d1",
+          "reimplements": "c125dab7-2a53-492f-a99a-56ad511940d8",
+          "comments": [
+            "Reimplemented to be logically consistent"
+          ],
+          "description": "A student can only be added to the same grade in the sorted roster more than once",
```

```suggestion
          "description": "A student can only be added to the same grade in the roster once",
```

- Whether or not the roster is sorted is irrelevant here.
- A student _cannot_ be added more than once.